src/pages/RadicalCollaboration/index.js

```diff
@@ -83,12 +30,95 @@ const RadicalCollaboration = () => {
       </Box>
       <Box className='containerGray'>
         <Container>
-          <Hidden mdDown>
-            <PhotoCardDesktop items={Images} />
-          </Hidden>
-          <Hidden lgUp>
-            <PhotoCardMobile items={Images} />
-          </Hidden>
+          <Grid container item>
+            <Grid>
+              <Paper elevation={0} className={classes.innerTextCardContainer}>
+                <Typography className={classes.cardHeading} variant='h4'>Donate </Typography>
+                <Typography variant="h6" color='primary' className={classes.subcardHeading}>Every gift helps us continue our work.</Typography>
+                <Typography className={classes.dtpLine1}>
+                 Your tax-deductible gift today will help make more open-source solutions easily available for the
+                 communities who need it the most. Thank You
+                </Typography>
+                <Button component={Link} to="/donate"  color='textSecondary' className={classes.donateButton}>
```

Comment:
color='textSecondary' is giving warning.

Reply:
I fixed those